expo/expo
# [shared-element] Update react-native-shared-element to 0.8.2

# Why

Updates `react-native-shared-element` to version `0.8.2`. This fixes various rendering issues on both Android and iOS, as well as adds support for React Native 0.65.

https://github.com/IjzerenHein/react-native-shared-element/releases

# How

- `et update-vendored-module --module react-native-shared-element --commit main`
- Updated Shared Element test in `test-suite` (to `react-navigation-shared-element@3.1.2`)
- Ran `npx pod install` in `apps/bare-expo/ios`
- Updated global `CHANGELOG.md`

# Test Plan

- Build and ran Expo Go on iOS (using XCode 12.5.1) (see video below)
- Build and ran Expo Go on Android Emulator (Pixel 2, API 30) (see video below)

**shared-element in test-suite on iOS**

https://user-images.githubusercontent.com/6184593/131666541-4be55b8f-98b6-4df4-889e-db7daa4eeca3.mp4

** shared-element in test-suite on Android**

https://user-images.githubusercontent.com/6184593/131828291-68cf95a3-af25-490c-af4c-60b526390a5b.mp4
